rocAL/rocAL/rocAL_hip/box_encoder_hip.cpp

```diff
@@ -249,7 +249,7 @@ void BoxEncoderGpu::Run(pMetaDataBatch full_batch_meta_data, float *encoded_boxe
         auto sample = &_samples_host_buf[sample_idx];
         //sample->in_box_count = full_batch_meta_data->get_bb_labels_batch()[sample_idx].size();
         HIP_ERROR_CHECK_STATUS( hipMemcpyHtoDAsync((void *)boxes_in_temp, full_batch_meta_data->get_bb_cords_batch()[sample_idx].data(), sample->in_box_count*sizeof(float)*4, _stream));
```

Comment:
Is the input bboxes in float format? I thought bbox struct is storing it in double. In this code it is expecting in float

Reply:
?